homeassistant/components/kef/media_player.py

```diff
@@ -69,6 +62,7 @@
         vol.Optional(
             CONF_INVERSE_SPEAKER_MODE, default=DEFAULT_INVERSE_SPEAKER_MODE
         ): cv.boolean,
+        vol.Optional(CONF_SUPPORTS_ON_OFF, default=DEFAULT_SUPPORTS_ON_OFF): cv.boolean,
```

Comment:
Is it impossible to detect this automatically?

Reply:
I don't think we can distinguish between Unavailable and this not being supported.

I also don't know how to get the serial number.

Reply:
Ok. So there's no documentation for the api?

Reply:
No, it is all reverse engineered at the byte level 😅

Reply:
Ok.